Seatbelt/Commands/Windows/InternetSettingsCommand.cs

```diff
@@ -54,6 +54,89 @@ public InternetSettingsCommand(Runtime runtime) : base(runtime)
+            {
+                foreach (var kvp in zoneMapKey)
+                {
+                    yield return new InternetSettingsDTO()
+                    {
+                        Hive = "HKLM",
+                        Key = kvp.Key,
+                        Value = zoneMapKeys.AsEnumerable().Single(l => l.Key == kvp.Value.ToString()).Value
+                    };
+                }
+            }
+
+            WriteHost("");
+
+            var zoneMapKey2 = RegistryUtil.GetValues(RegistryHive.CurrentUser, @"Software\Policies\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMapKey");
+
+            if ((zoneMapKey2 != null) && (zoneMapKey2.Count != 0))
+            {
+                foreach (var kvp in zoneMapKey2)
+                {
+                    yield return new InternetSettingsDTO()
+                    {
+                        Hive = "HKCU",
+                        Key = kvp.Key,
+                        Value = zoneMapKeys.AsEnumerable().Single(l => l.Key == kvp.Value.ToString()).Value
+                    };
+                }
+            }
+
+            WriteHost("");
+
+            // List Zones settings with automatic logons
+
+            /**
+             * HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Zones\{0..4}\1A00
+             * Logon setting (1A00) may have any one of the following values (hexadecimal):
+             * Value    Setting
+             *  ---------------------------------------------------------------
+             * 0x00000000 Automatically logon with current username and password
+             * 0x00010000 Prompt for user name and password
+             * 0x00020000 Automatic logon only in the Intranet zone
+             * 0x00030000 Anonymous logon
+            **/
+
+            WriteHost("Zone settings");
+            IDictionary<uint, string> zoneAuthSettings = new Dictionary<uint, string>()
+                                            {
+                                                {0x00000000, "Automatically logon with current username and password"},
+                                                {0x00010000, "Prompt for user name and password"},
+                                                {0x00020000, "Automatic logon only in the Intranet zone"},
+                                                {0x00030000, "Anonymous logon"}
+                                            };
+
+            for (int i = 0; i <= 4; i++)
+            {
+                var zoneSettings = RegistryUtil.GetDwordValue(RegistryHive.LocalMachine, @"Software\Policies\Microsoft\Windows\CurrentVersion\Internet Settings\Zones\" + i.ToString(), "1A00");
+                if (zoneSettings != null)
+                {
+                    WriteHost(zoneMapKeys.AsEnumerable().Single(l => l.Key == i.ToString()).Value + "\tSettings: " + zoneAuthSettings.AsEnumerable().Single(l => l.Key == zoneSettings).Value);
```

Comment:
@flacostec any chance you could remove the call to WriteHost and replace it with a TextFormatter similar to this: 
https://github.com/GhostPack/Seatbelt/blob/master/Seatbelt/Commands/Windows/OSInfoCommand.cs#L140-L180

Reply:
Actually, nevermind, working on it right now

Reply:
Okay good, let me know if you want me to do it.